src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py

```diff
@@ -51,10 +51,26 @@ def __init__(self, operation: tf.Operation, inner_graph: bool):
                                                     "Expected tf.Operation, got {}".format(type(operation))
         self.m_operation = operation
         self.m_inner_graph = inner_graph
+        self.m_shared_memory = True
         if self.m_operation.type == "Const":
+            # Copies tensor value to parsed TensorProto
             value = self.m_operation.node_def.attr["value"].tensor
-            # copies tensor value from node_def
-            self.m_parsed_content = tf.make_ndarray(value)
+
```

Comment:
Also, I think before sharing constant/variable values, we need to provide ovc/mo option for switching on/off sharing?
Besides, what mechanism cares that `tf.Variable` will stay in the memory while `ov::Model` is alive

Reply:
Yes, I can provide the option for switching on/off sharing, but the name of the option is not determined yet. So probably we can add it in separate PR when the name is clear and not block this PR?

This mechanism does not cares that `tf.Variable` still alive.
@slyalin mentioned that we can recommend user to keep original model in memory as we do with PyTorch FE. 
Or we can recommend to turn off sharing when the option is ready.